iwyu_config.h.in

```diff
@@ -0,0 +1,16 @@
+//===--- iwyu_confi.h - iwyu build configurations -------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef IWYU_CONFIG_H_IN
```

Comment:
This should probably have conventional IWYU header guards: `INCLUDE_WHAT_YOU_USE_IWYU_CONFIG_H_`